protocols/virtualfund/virtualfund.go

```diff
@@ -50,13 +52,77 @@ func (c *Connection) Equal(d *Connection) bool {
 	if !c.Channel.Equal(d.Channel) {
 		return false
 	}
+	//
+	// if c.ConsensusChannel.Id != d.ConsensusChannel.Id {
+	// 	return false
+	// }
```

Comment:
✂️

Reply:
addressed in 3a4fe9448ce10f290508d699d6fd6670036a79fa